README.md

```diff
@@ -1,6 +1,44 @@
 # Liberty Tools for VS Code
 
-[![Marketplace Version](https://vsmarketplacebadge.apphb.com/version/Open-Liberty.liberty-dev-vscode-ext.svg "Current Release")](https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext)
+## Building the Liberty Language Server Prototype
```

Comment:
Do we want to keep this on the main README? Maybe it should be moved to a separate doc. Keep in mind that the README is used on the VS Code Marketplace extension page

Reply:
When you say the README is used on the Marketplace extension page, do you mean it's using a direct link to the `main` branch README and not a static copy somewhere?

Reply:
I think it's a direct link...